_includes/links.md

```diff
@@ -11,9 +12,15 @@
 [cran-knitr]: https://cran.r-project.org/package=knitr
 [cran-stringr]: https://cran.r-project.org/package=stringr
 [dc-lessons]: http://www.datacarpentry.org/lessons/
+[diagnostic]: https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/diag_scripts/examples/diagnostic.py
+[ds]: https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/data/dataset.urls
 [email]: mailto:team@carpentries.org
 [github-importer]: https://import.github.com/
+[gitter-badge]: https://badges.gitter.im/ESMValGroup/Tutorial.svg
+[gitter]: https://gitter.im/ESMValGroup/Tutorial?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
```

Comment:
These links aren't needed here (since they stayed in the `README.md`)

Reply:
5e35e30 done